jobs/furnishings/src/furnishings/stage_processors/stage_one.py

```diff
@@ -95,6 +114,58 @@ def generate_paper_letters(self):
             self._app.logger.error(f'Error generating batch letters: {e}')
         self._app.logger.debug('Finish generating batch letters.')
 
+    def upload_to_sftp(self, client, data, filename):
+        """SFTP data to targeted destination."""
+        return client.putfo(
+            fl=BytesIO(data),
+            remotepath=f'{self._app.config.get("BCMAIL_SFTP_STORAGE_DIRECTORY")}/{filename}'
+        )
+
+    def update_notes_and_status(self, furnishings_list, funishing_status, furnishing_notes=None):
+        """Update the notes and status of furnishing entries in a list."""
+        for furnishing in furnishings_list:
+            furnishing.notes = furnishing_notes
+            furnishing.status = funishing_status
+            furnishing.save()
+
+    def process_paper_letters(self):
+        """Process the generated paper letts of BC and XPRO businesses (SFTP)."""
+        # Skip SFTPing PDF files to BCMail+ if flag is on
+        if self._disable_bcmail_sftp:
+            self._app.logger.debug(f'disable-dissolution-sftp-bcmail flag on: {self._disable_bcmail_sftp}')
+            return
+
+        current_date = datetime_util.now()
+        if self._bc_mail_furnishings:
+            try:
+                filename = current_date.strftime('DIS1_LETTER_BC_%b.%d.%Y.pdf')
+                # SFTP BC batch letter PDF to BCMail+
+                with self._bcmail_sftp_connection as client:
+                    resp = self.upload_to_sftp(client, self._bc_letters, filename)
+                self._app.logger.debug(f'Successfully uploaded {resp.st_size} bytes to BCMAIL+ SFTP (BC letter)')
+            except Exception as err:
+                self.update_notes_and_status(
+                    self._bc_mail_furnishings,
+                    Furnishing.FurnishingStatus.FAILED,
+                    'SFTP error of BC batch letter.'
+                )
+                self._app.logger.debug(f'SFTP error of BC batch letter: {err}')
+
+        if self._xpro_mail_furnishings:
+            try:
+                filename = current_date.strftime('DIS1_LETTER_EP_%b.%d.%Y.pdf')
```

Comment:
The screenshot for the naming of Xpro batch letter in the emails I was involved is `DIS1_LETTER_XPRO_<param_id_2>.pdf` instead of using `EP` (I know miro board uses both of them). Can you confirm with Dylan which one should be used or what is using in COLIN?

(Probably not a big deal, just want to be careful enough and the potential issue won't be raised on our side...

Reply:
Good catch. So according to Dylan, it doesn't matter. He told me to pick the one that I like 😆 I'll just leave as is I guess.